doc/nrf/doc_styleguide.rst

```diff
@@ -15,40 +15,105 @@ RST style guide
+| DK Button and LED           | DK Button and LED library        |
++-----------------------------+----------------------------------+
+
+Headings
+--------
+
+Headings use sentence case, which means that the first word is capitalized, and the following words use normal capitalization.
+The only exception are proper names, for example Bluetooth specification terminology (see last entry in the following table).
+
+.. sentence_case_table:
+
++-----------------------------------------------------+---------------------------------------------------+
+| Correct heading                                     | Incorrect heading                                 |
++=====================================================+===================================================+
+| CC file format                                      | CC File Format                                    |
++-----------------------------------------------------+---------------------------------------------------+
+| Sending shell commands                              | Sending Shell Commands                            |
++-----------------------------------------------------+---------------------------------------------------+
+| GATT Human Interface Device (HID) Service Client    | GATT Human Interface Device (HID) service client  |
++-----------------------------------------------------+---------------------------------------------------+
+
+Do not use consecutive headings without intervening text.
+
+RST file formatting
+===================
+
+For readability reasons, start every sentence on a new line in the source files and do not add line breaks within the sentence.
+In the output, consecutive lines without blank lines in between are combined into one paragraph.
+
+.. note:: For the conceptual documentation written in RST, you can have more than 80 characters per line.
+          The requirement for 80 characters per line applies only to the code documentation written in doxygen.
+
+The sentences must not end with a space.
+Trim trailing spaces before committing your changes.
+
+Each RST file must end with one blank line.
+
+Content highlighting
+====================
+
+Use the following highlighing rules in the RST conceptual documentation:
+
+.. content_highlighting_table:
+
++------------------------+------------------------+---------------------------------------------------------+
+| Markup                 | Example                | Usage criteria                                          |
++========================+========================+=========================================================+
+| ``*emphasis*``         | *relaying*             | Emphasis or new terms.                                  |
++------------------------+------------------------+---------------------------------------------------------+
+| ````code````           | ``west update``        | Code within text, including file paths and names.       |
```

Comment:
Actually, we also have the :file:`xxx` command. Do we want to establish any rules for when to use this?
I usually try to use it for file names or paths that include the file name, but not for directories. But surely not consistently.

Reply:
Use :file: for anything related to files - file names, file paths, directories, extensions.

Reply:
Updated table as discussed during the NCS sync meeting.